x-pack/plugin/ilm/qa/with-security/build.gradle

```diff
@@ -1,32 +1,16 @@
 apply plugin: 'elasticsearch.java-rest-test'
+apply plugin: 'elasticsearch.authenticated-testclusters'
 
 dependencies {
   javaRestTestImplementation(testArtifact(project(xpackModule('core'))))
 }
 
-def clusterCredentials = [
-        username: providers.systemProperty('tests.rest.cluster.username')
-                .orElse('test_admin')
-                .forUseAtConfigurationTime()
-                .get(),
-        password: providers.systemProperty('tests.rest.cluster.password')
-                .orElse('x-pack-test-password')
-                .forUseAtConfigurationTime()
-                .get()
-]
-
-tasks.named("javaRestTest").configure {
-  systemProperty 'tests.rest.cluster.username', clusterCredentials.username
-  systemProperty 'tests.rest.cluster.password', clusterCredentials.password
-}
-
 testClusters.all {
   testDistribution = 'DEFAULT'
-  setting 'xpack.security.enabled', 'true'
+  setting 'xpack.watcher.enabled', 'false'
```

Comment:
This line duplicates the following line.

Reply:
fixed